site-scrapers/NatickMall.js

```diff
@@ -58,7 +58,12 @@ async function ScrapeWebsiteData() {
 
     const availabilityResponse = await availabilityPromise;
     const availability = JSON.parse(availabilityResponse);
-    const results = { availability: {}, hasAvailability: false };
+    const results = {
+        availability: {},
+        hasAvailability: false,
+        timestamp: new Date(),
```

Comment:
timestamp goes at the top level, not here

Reply:
AH nevermind I see what you did. For consistency, could you add it at the top of this file? See a couple other scrapers for examples.